types/yeoman-generator/index.d.ts

```diff
@@ -6,12 +6,14 @@
 //                 Joshua Cherry <https://github.com/tasadar2>
 //                 Arthur Corenzan <https://github.com/haggen>
 //                 Richard Lea <https://github.com/chigix>
+//                 Devid Farinelli <https://github.com/misterdev>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import { EventEmitter } from 'events';
 import * as inquirer from 'inquirer';
 import { Observable } from 'rxjs';
+import { DiffieHellman } from 'crypto';
```

Comment:
Not needed I think?

Reply:
vscode auto-imported this and I didn't notice. Thank you! :)